package.json

```diff
@@ -36,8 +36,8 @@
   },
   "devDependencies": {
     "@types/jsan": "^3.1.5",
-    "expo": "~51.0.18",
-    "expo-module-scripts": "^3.5.2",
+    "expo": "~52.0.3",
+    "expo-module-scripts": "^4.0.2",
```

Comment:
would be good to bump the peerDependencies because after the bump, it supported sdk 52 or newer only:
e.g.
https://github.com/expo/dev-plugins/blob/305c6a6a12fcb391d303cd3353526bae5f0c8f1c/packages/apollo-client/package.json#L46